util/libcrypto.num

```diff
@@ -4501,3 +4501,28 @@ EVP_sha512_256                          4442	1_1_1	EXIST::FUNCTION:
 EVP_sha512_224                          4443	1_1_1	EXIST::FUNCTION:
 OCSP_basic_sign_ctx                     4444	1_1_1	EXIST::FUNCTION:OCSP
 RAND_DRBG_bytes                         4445	1_1_1	EXIST::FUNCTION:
+X509_LOOKUP_meth_new                    4446	1_1_1	EXIST::FUNCTION:
```

Comment:
IF this is intended to be backported to 1.1.0, the version MUST match the next 1.1.0 update, i.e. `1_1_0h`, for all these functions.